pom.xml

```diff
@@ -338,7 +338,7 @@
         </repository>
         <repository>
             <id>rubygems-releases</id>
-            <url>http://rubygems-proxy.torquebox.org/releases</url>
+            <url>https://rubygems-proxy.torquebox.org/releases/url</url>
```

Comment:
I think this change was part of your other pull request. Try `git pull --rebase master` on your PR branch and then force push with `git push -f` so that we can keep this PR focused on the XSS fix only.